docs/reference/execution-providers/OpenVINO-ExecutionProvider.md

```diff
@@ -107,7 +107,8 @@ Key-Value pairs for config options can be set using the Session.set_providers AP
 session = onnxruntime.InferenceSession(<path_to_model_file>, options)
 session.set_providers(['OpenVINOExecutionProvider'], [{Key1 : Value1, Key2 : Value2, ...}])
 ```
-*Note that this causes the InferenceSession to be re-initialized, which may cause model recompilation and hardware re-initialization*
+*Note that the next release (ORT 1.10) will require explicitly setting the providers parameter if you want to use execution providers other than the default CPU provider (as opposed to the current behavior of providers getting set/registered by default based on the build flags) when instantiating InferenceSession.*
+*Note that this causes the InferenceSession to be re-initialized, which may cause model recompilation and hardware re-initialization.*
```

Comment:
The comment about InferenceSession to be re-initialized applies to the set_providers() example above? I think that would cause confusion.

I think it would be better to change the example above use OpenVINOExecutionProvider for the InferenceSession creation and remove the set_providers() call

Reply:
Okay, let me rewrite the example code.